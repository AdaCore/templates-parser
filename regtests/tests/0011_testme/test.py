from test_support import *

run('testme', ["testme17.tmplt"])
run('print_tree', ["testme17.tmplt"])
