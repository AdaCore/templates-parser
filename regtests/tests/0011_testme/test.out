
  OK

  OK

  OK

  OK

  OK

[INFO] testme17.tmplt
[C_INFO]  1 FALSE
[TEXT] 
[IF] (@_VAR1_@ = "a value")
  [TEXT]   OK
[ELSE]
  [TEXT]   NOK
[END_IF]
[TEXT] 
[IF] (@_VAR1_@ /= value)
  [TEXT]   OK
[ELSE]
  [TEXT]   NOK
[END_IF]
[TEXT] 
[IF] (@_VAR1_@ /= value)
  [TEXT]   OK
[ELSE]
  [TEXT]   NOK
[END_IF]
[TEXT] 
[IF] ("a value" = @_VAR1_@)
  [TEXT]   OK
[ELSE]
  [TEXT]   NOK
[END_IF]
[TEXT] 
[IF] ("toto value" /= @_VAR1_@)
  [TEXT]   OK
[ELSE]
  [TEXT]   NOK
[END_IF]
